### PR TITLE
appmesh pipeline - added test route and outputs

### PR DESCRIPTION
**Pull request by srinivas-toluna**
appmesh pipeline related changes

### CODE REVIEW

File: main.tf [hunk 1]
```diff
@@ -216,6 +257,38 @@ resource "aws_appmesh_route" "integrators" {
...
+#   # Ignoring changes made by code_deploy controller
+#   lifecycle {
+#     ignore_changes = [
+#       spec[0].http_route[0].action
+#     ]
+#   }
+# }
 
 
 resource "aws_appmesh_virtual_node" "td_net" {
```
Comment (ShakedYosef) [inline main.tf]: What is "td_net"? please find clear & generic name.

File: main.tf [hunk 2]
```diff
@@ -216,6 +257,38 @@ resource "aws_appmesh_route" "integrators" {
   }
 }
 
+# resource "aws_appmesh_route" "test_route" {
```
Comment (ShakedYosef) [inline main.tf]: If these lines (which are in comment) are not necessary, please delete them.

File: main.tf (hunk 1 above)
Comment (srinivas-toluna) [reply main.tf]: This is part module that existed. Not created now.

File: main.tf (hunk 2 above)
Comment (srinivas-toluna) [reply main.tf]: done

**Review by ShakedYosef — APPROVED** [review]

**Review by Eli-Meitner — APPROVED** [review]

**Pull request merged**